keyboards/nightmare/rules.mk

```diff
@@ -0,0 +1,80 @@
+# MCU name
+MCU = atmega32u4
```

Comment:
Actually, all of the stuff between MCU and BOOTLOADER are no longer necessary, and are specified by default.   So they can be removed, safely.